runelite-client/src/main/java/net/runelite/client/ui/overlay/infobox/InfoBox.java

```diff
@@ -26,23 +26,35 @@
 
 import java.awt.Color;
 import java.awt.image.BufferedImage;
+import net.runelite.client.plugins.Plugin;
 
 public abstract class InfoBox
 {
 	private final BufferedImage image;
 
+	private final Plugin plugin;
```

Comment:
Instead of passing entire plugin here, pass only plugin's unique identifier, e.g PluginDescriptor